src/sentry/api/serializers/models/incident.py

```diff
@@ -46,7 +46,9 @@ def serialize(self, obj, attrs, user):
             "dateStarted": obj.date_started,
             "dateDetected": obj.date_detected,
             "dateCreated": obj.date_added,
-            "dateClosed": obj.date_closed,
+            "dateClosed": obj.date_closed.replace(second=0, microsecond=0)
+            if obj.date_closed
+            else None,
```

Comment:
feels like black doesn't really know what to do with this ternary

Reply:
Yeah this is a bit gross, maybe i'll clean it up